docs/index.rst

```diff
@@ -89,16 +95,22 @@ and any new tests you have written.::
 
 **Notes:**
 
-- CLI tests run easiest if you include the root credentials in robottelo.properties
+- CLI tests run easiest if you include the root credentials in server.yaml
 - UI tests should be configured to run through your SauceLabs account.
 
 Running the Tests
 =================
 
 Before running any tests, you must create a configuration file::
 
-    $ cp robottelo.properties.sample robottelo.properties
-    $ vi robottelo.properties
+    $ cp broker/path/broker_settings.yaml ./broker_settings.yaml
+    $ vi broker_settings.yaml
+    $ cp robottelo.yaml.sample ./robottelo.yaml
```

Comment:
```suggestion
    $ cp conf/robottelo.yaml.template conf/robottelo.yaml
```

Reply:
we can leave off the `conf/` prefix if we explicitly add a `cd conf` step before this section

Reply:
That file should be refactored soon, but don't worry about it for now.